DUI3-DX/Sdk/Speckle.Connectors.Utils/Operations/Send.cs

```diff
@@ -160,7 +159,7 @@ public static class SendHelper
     await Task.WhenAll(transportAwaits).ConfigureAwait(false);
 
     var parsed = JObject.Parse(obj);
-    JToken? idToken = parsed.GetValue("id");
+    JToken? idToken = parsed.GetValue("id", StringComparison.CurrentCultureIgnoreCase);
```

Comment:
What was the reason for this change?

Reply:
apparently, the overload for getting a value wants a string comparison to be accurate.   I probably should just do ordinal though